nullaway/src/main/java/com/uber/nullaway/handlers/CompositeHandler.java

```diff
@@ -238,11 +244,12 @@ public Optional<ErrorMessage> onExpressionDereference(
 
   @Override
   public boolean includeApInfoInSavedContext(AccessPath accessPath, VisitorState state) {
-    boolean shouldFilter = false;
     for (Handler h : handlers) {
-      shouldFilter |= h.includeApInfoInSavedContext(accessPath, state);
+      if (h.includeApInfoInSavedContext(accessPath, state)) {
+        return true;
+      }
```

Comment:
This one is safe right now, given current implementations of `includeApInfoInSavedContext`, but a) not worth breaking the general pattern and b) I can imagine handler implementations that set either instance state or `VisitorState` state based on this callback.

Reply:
ok removed